astroquery/alma/core.py

```diff
@@ -961,50 +887,3 @@ def unique(seq):
 def filter_printable(s):
     """ extract printable characters from a string """
     return filter(lambda x: x in string.printable, s)
-
-
-def parse_frequency_support(frequency_support_str):
```

Comment:
Yes. Also updated ``approximate_primary_beam_sizes`` in utils, to have all the same parameters as the one deleted from core.